src/creator.html

```diff
@@ -676,7 +678,8 @@ <h4>Manage This Hotspot</h4>
 						<!-- Background image placed behind the hotspot's SVG canvas -->
 						<img id="hotspotImage"
 							ng-src="{{image.src}}"
-							ng-show="mediaReady" />
+							ng-show="mediaReady && showImage" />
+						<iframe ng-show="mediaReady && !showImage" width="100%" height="100%" id="hotspotImage" ng-src="{{url}}" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Is this iframe still required for hotspots?

Reply:
oops. It shouldn't be, I'll remove it

Reply:
:+1: